src/Model/Forwarder.php

```diff
@@ -221,4 +223,40 @@ protected function createReviews($limit = null, $offset = null, array $uids = []
 
         return $reviews;
     }
+
+    /**
+     * @return array|mixed[]
+     */
+    public function createPluginInfo()
+    {
+        $objectManager   = ObjectManager::getInstance();
```

Comment:
you do not use object manager here, we only use it once, please see: `$forwarderInitializer = $manager->get('Shopgate\Base\Helper\Initializer\Forwarder');`
It just instantiates the needed classes and you just use a getter to load them. The reason being is that we cannot use DI here bcs of final constructor of the extended class. However, we want to use the object manager minimally, so we only call it once here. Once in the Config (for same reason) and once in your other PR.